Source/FieldSolver/SpectralSolver/SpectralFieldDataRZ.H

```diff
@@ -57,6 +60,12 @@ class SpectralFieldDataRZ
                                    amrex::MultiFab & tempHTransformedSplit,
                                    const bool is_nodal_z);
 
+        void InitFilter(amrex::IntVect const & filter_npass_each_dir, bool const compensation,
+                        SpectralKSpaceRZ const & k_space);
+
+        void ApplyFilter(int const field_index);
+        void ApplyFilter(int const field_index1, int const field_index2, int const field_index3);
```

Comment:
I would add a whitespace between function names and parentheses, as these are not function calls:
```suggestion
        void InitFilter (amrex::IntVect const & filter_npass_each_dir, bool const compensation,
                         SpectralKSpaceRZ const & k_space);

        void ApplyFilter (int const field_index);
        void ApplyFilter (int const field_index1, int const field_index2, int const field_index3);
```